opslevel/terraform_type_conversions.go

```diff
@@ -47,3 +47,13 @@ func unquote(value string) string {
 	}
 	return value
 }
+
+// Converts a basetypes.ListValue to a []string
+func ListValueToStringSlice(ctx context.Context, listValue basetypes.ListValue) ([]string, diag.Diagnostics) {
+	dataAsSlice := []string{}
+	if listValue.IsNull() {
+		return dataAsSlice, nil
+	}
+	diags := listValue.ElementsAs(ctx, &dataAsSlice, false)
```

Comment:
Had a question about this function for a while - what's the implication of passing `false` here instead of `true`?

Reply:
This flag determines how null values are handled. If `listValue` is a `null` value then setting this to `true` would automatically set `dataAsSlice` to an empty string slice. It's extra here since that check is handled explicitly above but just to set a precedent this can be set to `true` in case this is copy pasted elsewhere

Reply:
Thanks!